WalletWasabi/Helpers/Constants.cs

```diff
@@ -99,6 +99,8 @@ public static BitcoinWitPubKeyAddress GetCoordinatorAddress(Network network)
 
 		public const string Chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
 
+		public const float TransactionRBFSignalRate = 0.02F; // 2% RBF transactions
```

Comment:
Why did you use `0.02F` and not `0.02d`?

Reply:
That's just what my IDE told me to do

Reply:
Every number with a decimal point is double by default. To have float u must put F there. I would use double for simplicity and mainly because double compared with a double in the code, it is cleaner.